Sources/Classes/PDFViewController.swift

```diff
@@ -84,8 +84,17 @@ public final class PDFViewController: UIViewController {
     /// UIBarButtonItem used to override the default action button
     fileprivate var actionButton: UIBarButtonItem?
     
+    /// Default background color applied both to the view and the collectionView.
+    fileprivate var defaultBackgroundColor: UIColor {
+        return UIColor(red: 236/255, green: 239/255, blue: 241/255, alpha: 1)
+    }
+    
     override public func viewDidLoad() {
         super.viewDidLoad()
+        
+        self.view.backgroundColor = defaultBackgroundColor
+        
+        collectionView?.backgroundColor = defaultBackgroundColor
```

Comment:
I'm not sure why we need to access the ```collectionView``` optionally here? It would appear the collection view is an ```implicitly unwrapped optional```.